server/src/main/java/org/opensearch/node/remotestore/RemoteStoreNodeService.java

```diff
@@ -249,4 +250,12 @@ public static boolean isMigratingToRemoteStore(Metadata metadata) {
 
         return (isMixedMode && isRemoteStoreMigrationDirection);
     }
+
+    public RemoteStoreNodeStats getRemoteStoreNodeStats() {
+        long lastSuccessfulFetchOfPinnedTimestamps = 0;
+        if (RemoteStoreSettings.isPinnedTimestampsEnabled()) {
```

Comment:
Do we need to check this? If the setting is disabled, what is the value that is returned?